demo/meson.build

```diff
@@ -12,6 +12,7 @@ executable(
     'Views/DialogsView.vala',
     'Views/DynamicNotebookView.vala',
     'Views/FormView.vala',
+    'Views/HyperTextView.vala',
```

Comment:
To avoid a name clash (albeit in different namespaces) it may be better to call this `HyperTextViewDemo.vala` Not important I guess.

Reply:
Adressed the concerns by naming the view `HyperTextViewGrid`.